### ISSUE TITLE
Go through all Debt Team Sidekiq jobs and ensure they have exhaustion blocks

### ISSUE THREAD

**Issue by digitaldrk** [department-of-veterans-affairs/va.gov-team#91367]
## TL;DR
There have been instances in va.gov where vets submit a form or perform some action and there was an error and no one knows. We need to ensure we have exhaustion blocks in all of our jobs so we can be notified about the issue and rectify it.

## Tech Details
I met with Steve Albers today (8/23) and he mentioned how helpful queue exhaustion blocks **with alerts** are for catching silent issues.

One of the things on his todo list is to go through all Sidekiq jobs across the platform and ensure these are added. We should add these so we can keep tabs on issues that may go un-noticed.

We may want to break this card up into multiple cards that name specific Sidekiq jobs.

- [ ] Update jobs to use the same exhausted naming convention
- [ ] Create new monitors that look for failures in the past hour
- [ ] Create new Dashboards that use the new 'exhausted' naming convention

Here is an example:
https://github.com/department-of-veterans-affairs/vets-api/pull/17136/files

## Tasks
- [ ] Go through all Sidekiq jobs across the platform and ensure queue exhaustion blocks **with alerts** are added

## Design and Front End sync on implementation of design
- [ ] Required
- [ ] Complete
- [X] Not Required

## Acceptance criteria
- [ ] All Sidekiq jobs across the platform have queue exhaustion blocks **with alerts**

## Testing
- [x] N/A if non development work
- [ ] Testing passed and documented in this ticket based off the "Then" statement in the description

Testing procedure:
If additional testing steps or credentials to perform testing are needed list them here

---

## Ticket creation

<details><summary>Ticket requirements</summary>

### Required to be "Ready"
- [X] Description written
- [X] Tasks defined
- [X] Acceptance criteria written
- [ ] Testing defined
- [X] Add Labels
- [X] Attach to an Epic
- [X] Estimate Points
</details>

**Comment by Tom-Davis** [comment]
@digitaldrk can this be combined with the silent error work?

**Comment by JunTaoLuo** [comment]
Here are my ongoing investigation notes. I'm going off the .github/CODEOWNERS file to account for all of the code we own and found Sidekiq jobs in the following instances:

| Module  | File | Exhaustion Block Added | Log/StatsD Added | Monitor Added |
| - | - | - | - | - |
| app/sidekiq/copay_notifications | mcp_notification_email_job.rb | no | no | no |
| app/sidekiq/copay_notifications | new_statement_notification_job.rb | yes | yes | no?|
| app/sidekiq/copay_notifications | parse_new_statements_job.rb | no | no | no |
| app/sidekiq/evss | document_upload.rb | no | no | no |
| lib/debt_management_center | va_notify_email_job.rb | no | no | no |
| modules/debts_api | form5655_submission.rb | yes | no | no |
| modules/debts_api | vba_submission_job.rb | yes | no | no |
| modules/debts_api | vha_submission_job.rb | yes | no | no |
| modules/debts_api | sharepoint_submission_job.rb | yes | yes | no |
| modules/debts_api | vbs_submission_job.rb | yes | yes | no |
| modules/debts_api | financial_status_report_service.rb | no | no | no |

**Comment by JunTaoLuo** [comment]
FYI this work was bigger than expected and will require two PRs to complete:
1. https://github.com/department-of-veterans-affairs/vets-api/pull/18472 this is Derek's PR and it is very close. We should have this merged today, or tomorrow at the latest
2. https://github.com/department-of-veterans-affairs/vets-api/pull/18675 this is my PR that needs to be rebased on Derek's PR once that's merged. It will take another day or two after that to go through the backend review process. I expect these to be wrapped up next sprint by end of the week.

**Comment by JunTaoLuo** [comment]
The work here has been completed, adding DD monitors will be tracked separately.
